cylc/flow/scheduler_cli.py

```diff
@@ -276,6 +274,28 @@ def scheduler_cli(parser, options, args, is_restart=False):
             # Prevent recursive host selection
             base_cmd.append("--host=localhost")
             return remote_cylc_cmd(base_cmd, host=host)
+    suite_srv_dir = suite_files.get_suite_srv_dir(reg)
+    keys = {
+        "client_public_key": KeyInfo(
+            KeyType.PUBLIC,
+            KeyOwner.CLIENT,
+            suite_srv_dir=suite_srv_dir, platform=host),
+        "client_private_key": KeyInfo(
+            KeyType.PRIVATE,
+            KeyOwner.CLIENT,
+            suite_srv_dir=suite_srv_dir),
+        "server_public_key": KeyInfo(
+            KeyType.PUBLIC,
+            KeyOwner.SERVER,
+            suite_srv_dir=suite_srv_dir),
+        "server_private_key": KeyInfo(
+            KeyType.PRIVATE,
+            KeyOwner.SERVER,
+            suite_srv_dir=suite_srv_dir)
+    }
+    # Clean any existing authentication keys and create new ones.
+    suite_files.remove_keys_on_server(keys)
+    suite_files.create_server_keys(keys, suite_srv_dir)
```

Comment:
Can this use the logic in `cylc/flow/tests/network/key_setup.py` to avoid duplication?

Reply:
Yep, I'll sort that.

Reply:
I have now removed this logic to cylc/flow/network/authenitcation.py to declutter scheduler_cli.